pyglui/ui_elements.pxi

```diff
@@ -929,25 +938,51 @@ cdef class Thumb(UI_element):
             utils.draw_circle(self.button.center,radius=int(min(self.button.size)*.77), stroke_width = stroke_width, color=RGBA(*thumb_color_shadow),sharpness=thumb_button_shadow_sharpness)
             utils.draw_circle(self.button.center,radius=int(min(self.button.size)*.77), stroke_width = stroke_width, color=self.off_color,sharpness=thumb_button_sharpness)
 
+        print 'button center: ', self.button.center
+        print 'radius: ', int(min(self.button.size)*.77)
+        print 'stroke_width: ', stroke_width
+
+
         if self.selected:
             self.selected = False
             global should_redraw
             should_redraw = True
 
         glfont.push_state()
+
+        # Choose either icon (fontawesome) or letter(roboto) label for button
+        cdef bytes text_to_draw
+        cdef float size_of_text
+        cdef list text_center_point
+
         glfont.set_font('roboto')
+        text_to_draw = self.label[0]
```

Comment:
Get rid of `text_to_draw` and always draw `self.label`. If people draw strings longer than one character, it is their fault.